bcda/cli.go

```diff
@@ -105,11 +109,15 @@ func importCCLF0(fileMetadata *cclfFileMetadata) (map[string]cclfFileValidator,
 					}
 					count, err := strconv.Atoi(string(bytes.TrimSpace(b[totalRecordStart:totalRecordEnd])))
 					if err != nil {
+						fmt.Printf("Failed to parse %s record count from CCLF0 file", filetype)
+						err = errors.Wrapf(err,"Failed to parse %s record count from CCLF0 file", filetype)
```

Comment:
"Failed..." message here is part of an error so we don't want that capitalized.

Reply:
doh! good catch